actions/run_action.go

```diff
@@ -68,6 +70,22 @@ func (run *RunAction) Verify(context *debos.DebosContext) error {
 	if run.Script == "" && run.Command == "" {
 		return errors.New("Script and Command both cannot be empty")
 	}
+
+	if run.Script != "" {
+		argv := strings.SplitN(run.Script, " ", 2)
+		script := debos.CleanPathAt(argv[0], context.RecipeDir)
+
+		stat, err := os.Stat(script)
+		if err != nil {
+			return err
+		}
+
+		/* check the script is executable */
+		if stat.IsDir() || stat.Mode()&0111 == 0 {
```

Comment:
what if the script is a symlink?

Reply:
Split the executable/dir check into two

it also works; clarified that test in the commit message